lib/bridge-client/blacklist.js

```diff
@@ -1,29 +1,25 @@
 'use strict';
 
-var assert = require('assert');
-var fs = require('fs');
-var utils = require('../utils');
-var path = require('path');
+var concat = require('concat-stream');
 
 /**
  * Manage a blacklist file containing an object with key value pairs of
  * nodeids: timestamp
  * @constructor
  * @license LGPL-3.0
  * @see https://github.com/storj/bridge
- * @param {String} options.blacklistFolder - blacklist folder location
  * @param {Object} options.logger - Logger instance
+ * @param {Object} options.store - The store that blacklist enteries will be
+ * persisted to. This object must be compatible with the API of
+ * [abstract-blob-store](https://github.com/maxogden/abstract-blob-store)
```

Comment:
For links in JSDoc comments, use `{@link https://github.com/maxogden/abstract-blob-store|abstract-blob-store}` to render it properly on the docs sits.